test/qu/test/query/validation.clj

```diff
@@ -66,10 +67,14 @@
       (does-contain (errors (q :limit "ten")) :limit)
       (does-not-contain (errors (q :limit "10")) :limit))
 
+    (testing "it errors if limit is greater than 100"
+      (does-contain (limit-errors (q :limit "101")) :limit)
+      (does-not-contain (limit-errors (q :limit "100")) :limit))
+
     (testing "it does not error if limit is greater than 1000"
```

Comment:
This is confusing... the new test checks for errors if the limit is greater than 100, but the existing test still checks for absence of error if limit is greater than 1000.